### DISCUSSION TITLE
YunoHost package for Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by jedie** [Koenkk/zigbee2mqtt#20799] [Show and tell]
Zigbee2MQTT is on the YunoHost App wishlist: https://apps.yunohost.org/wishlist?search=zigbee2mqtt and there is also https://github.com/Koenkk/zigbee2mqtt/issues/16516

I started to package it for YunoHost here: https://github.com/YunoHost-Apps/zigbee2mqtt_ynh

Discuss also here: https://forum.yunohost.org/t/yunohost-package-for-zigbee2mqtt/28039

A little boring experience is: The Zigbee2MQTT frontend will not start if there is no supported adapter accessible :(
This is also a problem in YunoHost package CI, because the tests will always ran into `502 Bad Gateway`, see: [YunoRunner for CI](https://ci-apps-dev.yunohost.org/ci/job/12683)

Any idea to avoid this?

It would be great if the frontend would be usable, but would indicate to the user that no working stick was found.

**Comment by jedie** [comment]
nobody a idea?

**Reply by abeorch** [reply]
Whats' the issue with the Package CI? - The link is broken.
